src/validator.cpp

```diff
@@ -1047,11 +1125,11 @@ bool Validator::ValidatorImpl::isCellmlIdentifier(const std::string &name)
     return result;
 }
 
-bool Validator::ValidatorImpl::unitsAreEquivalent(const ModelPtr &model,
-                                                  const VariablePtr &v1,
-                                                  const VariablePtr &v2,
-                                                  std::string &hints,
-                                                  double &multiplier)
+bool unitsAreEquivalent(const ModelPtr &model,
+                        const VariablePtr &v1,
+                        const VariablePtr &v2,
+                        std::string &hints,
+                        double &multiplier)
 {
```

Comment:
Probably not relevant to this PR, but what do you think about (later) removing these functions from the validator in favour of the equivalent() and compatible() ones in #554 ?

Reply:
Sounds like a great idea.